Language/Generics/Superbounded_types/well_bounded_A02_t04.dart

```diff
@@ -6,10 +6,9 @@
 /// compile-time error.
 ///
 /// @description Checks that it is a compile-time error to call a generic
-/// function with a type parameter which is not well-bounded
+/// function with a type parameter which is not regular-bounded
```

Comment:
This was one of those places where I suddenly realized that it's dangerous to talk about 'regular-bounded' (or even 'X-bounded' for any 'X'). Also, we often omit 'formal' vs. 'actual', but we normally use 'formal type parameter' or just 'type parameter' about the declaration vs. 'actual type argument' or just 'type argument' at the call site, and similarly for value parameters/arguments.
```suggestion
/// function with an actual type argument which does not satisfy the bounds.
```